ui/src/data_explorer/components/FieldListItem.tsx

```diff
@@ -116,28 +104,56 @@ class FieldListItem extends Component {
       </div>
     )
   }
-}
 
-const {string, shape, func, arrayOf, bool} = PropTypes
-
-FieldListItem.propTypes = {
-  fieldFuncs: arrayOf(
-    shape({
-      type: string.isRequired,
-      value: string.isRequired,
-      alias: string,
-      args: arrayOf(
-        shape({
-          type: string.isRequired,
-          value: string.isRequired,
-        })
-      ),
+  private toggleFunctionsMenu = (e: MouseEvent<HTMLElement>) => {
+    if (e) {
```

Comment:
Do you think this check might not be necessary now? Will there ever be a time when this doesn't have an event?